packages/aws-cdk-lib/aws-ec2/lib/instance-types.ts

```diff
@@ -1260,6 +1320,11 @@ export enum InstanceSize {
    */
   XLARGE56 = '56xlarge',
 
+  /**
+   * Instance size XLARGE56 (96xlarge)
+   */
+  XLARGE96 = '96xlarge',
+
   /**
    * Instance size XLARGE56 (112xlarge)
```

Comment:
can you fix this while you're in here? `XLARGE56`

Reply:
Done, thanks